pkg/informer/informer.go

```diff
@@ -44,6 +101,154 @@ func InformerForResource(res schema.GroupVersionResource) (GenericInformer, erro
 	return i, nil
 }
 
+func isResourceAllowed(cm *v1.ConfigMap, group, kind string, allowedList []AllowedResources, deniedList []DeniedResources) bool {
+
+	// fmt.Println(kind, group)
+	var boolVar bool
+
+	list := []string{"events", "projects", "clusters", "clusterstatuses", "oauthaccesstokens"}
+
+	//remove all apiResources with kind in list
+	for _, name := range list {
+		if kind == name {
+			fmt.Println("list of names, resource kind:", name, kind)
+			boolVar = false
```

Comment:
Need to `return false` as soon as we hit this condition, otherwise the value of `boolVar` can get changed later.